src/main/resources/assets/mekanism/lang/ja_jp.lang

```diff
@@ -1136,6 +1142,10 @@ mekanism.configgui.ctgy.tools.general.tooltip=MekanismToolsの基本設定
 mekanism.configgui.ctgy.tools.armor.tooltip=MekanismToolsの防具におけるパラメータの設定
 mekanism.configgui.ctgy.tools.tools.tooltip=MekanismToolsの道具におけるパラメータの設定
 
+# Commands

```

Comment:
The different command lang files added here should have a value on the right side of the equals, or just not be included.

Reply:
Fixed.

Reply:
It appears to still have the "blank" entries I was talking about:
```
cmd.mek.usage=
cmd.mek.debug.usage=
cmd.mek.debug=
cmd.mek.chunk.usage=
```

Reply:
I don't know what value to write there. The entry field seems to be blank for other translations. Can't I leave this blank?

Reply:
Ah, I had not noticed/realized they are blank even in the main en_us.lang file. This is fine as is then and I will figure out what to do with them separately. Is this good to merge or are you still making other misc changes.